src/speculators/data_generation/preprocessing.py

```
@@ -97,6 +98,9 @@ def _normalize_conversation(
         if "thinking" in turn and turn["thinking"]:
             normalized_turn["thinking"] = turn["thinking"]
 
+        if "reasoning_content" in metadata and metadata["reasoning_content"]:
```

Ruff (0.15.13)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)